ansible_collections/arista/avd/examples/single-dc-l3ls/group_vars/FABRIC.yml

```
@@ -47,5 +47,39 @@ bgp_peer_groups:
 
 # P2P interfaces MTU, includes VLANs 4093 and 4094 that are over peer-link
 # If you're running vEOS-lab or cEOS, you should use MTU of 1500 instead as shown in the following line
-# p2p_uplinks_mtu: 1500
+# p2p_uplinks_mtu: 9214
 p2p_uplinks_mtu: 1500
+
+# Set default uplink, downlink, and MLAG interfaces based on node type
+default_interfaces:
+  - types: [ spine ]
+    platforms: [ default ]
+    uplink_interfaces: [ Ethernet1-2 ]
+    downlink_interfaces: [ Ethernet1-8 ]
+  - types: [ l3leaf ]
+    platforms: [ default ]
+    uplink_interfaces: [ Ethernet1-2 ]
+    mlag_interfaces: [ Ethernet3-4 ]
+    downlink_interfaces: [ Ethernet8 ]
+  - types: [ l2leaf ]
+    platforms: [ default ]
+    uplink_interfaces: [ Ethernet1-2 ]
+
+# CloudVision Portal definitions
+cvp_instance_ip: 192.168.1.12
+terminattr_smashexcludes: "ale,flexCounter,hardware,kni,pulse,strata"
+terminattr_ingestexclude: "/Sysdb/cell/1/agent,/Sysdb/cell/2/agent"
+terminattr_disable_aaa: true
+
+# DNS Server
+name_servers:
+  - 192.168.1.1
+
+# NTP Servers IP or DNS name, first NTP server will be prefered, and sourced from Management VRF
+custom_structured_configuration_ntp:
```

I don't remember why we did this?
```suggestion
ntp:
```

---

Maybe it was to get the config into the structured_config

---

Yes, without custom_structure_configuration_ntp it doesn't work.